jabls/build.gradle.kts

```diff
@@ -16,6 +16,8 @@ dependencies {
 
     implementation("com.google.guava:guava")
 
+    implementation("org.hisp.dhis:json-tree")
```

Comment:
you need to add this module to org.jabref.gradle.base.dependency-rules.gradle.kts as well